packages/nimble-components/build/generate-workers/source/render-worker.ts

```diff
@@ -0,0 +1,44 @@
+import * as Comlink from 'comlink';
+import { HealthStatus, healthStatus } from './health-status';
+
+export class RenderWorker {
+    private worker!: number;
+    private dieMatrix: Uint8Array = Uint8Array.from([]);
+    private performanceTest: string | undefined;
```

Comment:
When is this not `undefined`?

Reply:
We wanted to provide some more accurate performance tests, but after talking with @munteannatan I removed this part